spec/spec_helper.rb

```diff
@@ -56,6 +56,8 @@ def format(result)
   # config.mock_with :flexmock
   # config.mock_with :rr
 
+  config.full_backtrace = true
```

Comment:
@cbeer I don't care. It seems like it's helpful for debugging.  Do you want it removed?